website/benchmarks.html

```diff
@@ -153,10 +156,25 @@ <h3 id="syscalls">Syscall count <a href="#syscalls">#</a></h3>
     <script type="module">
       import { drawCharts } from "./app.js";
       window.chartWidth = 800;
+      const busy = document.getElementById("busy")
 
-      let u = window.location.hash.match("all") ? "./data.json" : "recent.json";
+      function showSpinner () {
+        busy.style.display = "block";
+      }
 
-      drawCharts(u);
+      function hideSpinner () {
+        busy.style.display = "none";
+      }
+
+      function updateCharts () {
+        let u = window.location.hash.match("all") ? "./data.json" : "recent.json";
```

Comment:
How about making `u` `const`?

Reply:
I agree (I didn't add that line).

I see a lot of js code using `let` where `const` could (should ?) be used.

I always favour using `const` and rarely use `let` but others may disagree.

I'll make the change.